main.go

```diff
@@ -910,7 +911,28 @@ func sidekickMain(ctx *cli.Context) {
 
 	router.PathPrefix(slashSeparator).Handler(m)
 	if ctx.String("cert") != "" && ctx.String("key") != "" {
-		if err := http.ListenAndServeTLS(addr, ctx.String("cert"), ctx.String("key"), router); err != nil {
+		manager, err := certs.NewManager(context.Background(), ctx.String("cert"), ctx.String("key"), tls.LoadX509KeyPair)
+		if err != nil {
+			console.Fatalln(err)
+		}
+		tlsConfig := &tls.Config{
+			PreferServerCipherSuites: true,
+			NextProtos:               []string{"h2", "http/1.1"},
```

Comment:
Do we really prefer http/2?

Reply:
I was asking that myself too, but I found exactly the same code in the Minio repo, so I guess it was copied from there.

Reply:
Now I want to refer to `cmd/utils.go:952` again, I see the order is different...

Reply:
Yesh. From console with 
```
			// Use modern tls mode https://wiki.mozilla.org/Security/Server_Side_TLS#Modern_compatibility
			NextProtos: []string{"h2", "http/1.1"},
```
But I check the go code here. Just use strSliceContains here. I think order is ok. @ramondeklein @klauspost 
@harshavardhana I don't know too.

Reply:
on MinIO we prefer http/1.1 first sidekick should also have `http/1.1` first. http2 is great but Go's implementation is shoddy and riddled with bugs.